aptos-move/aptos-release-builder/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "aptos-release-builder"
```

Comment:
Do you plan to expand the scope of the tool in the future? Otherwise maybe it's better to have a more specific name

Reply:
Maybe `aptos-release-governance-tool`?

Reply:
aptos-governance-proposal-generator?